twilight-http/src/request/guild/sticker/update_guild_sticker.rs

```diff
@@ -153,12 +161,13 @@ impl IntoFuture for UpdateGuildSticker<'_> {
 
 impl TryIntoRequest for UpdateGuildSticker<'_> {
     fn try_into_request(self) -> Result<Request, Error> {
-        let request = Request::builder(&Route::UpdateGuildSticker {
+        let fields = self.fields.map_err(Error::validation)?;
+
+        Request::builder(&Route::UpdateGuildSticker {
             guild_id: self.guild_id.get(),
             sticker_id: self.sticker_id.get(),
         })
-        .json(&self.fields)?;
-
-        Ok(request.build())
+        .json(&fields)
+        .build()
     }
 }
```

Comment:
**Observation**: We don't use the reason here which I think is a bug, but may be out of scope of this pr.

Reply:
https://github.com/twilight-rs/twilight/pull/2181